backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1042,3 +1042,16 @@ databaseChangeLog:
               - column:
                   name: latest_recorded_timestamp
                   descending: true
+  - changeSet: # NOTE: this has not been tested
+      id: change-test-results-to-MST
+      author: neil.s.sharma@omb.eop.gov
+      comment: changes all existing test results to be MST ISO 8601 strings
+      changes:
+        - sql:
+            comment: Update all completed test_orders from UTC to MST. This is a one-time update.
+            sql: |
+              UPDATE ${database.defaultSchemaName}.test_order
+              SET date_tested_backdate = (
+                SELECT date_tested_backdate at time zone 'utc' at time zone 'mst'
+                WHERE order_status='COMPLETED'
+              );
```

Comment:
There are two columns in the `test_order` table:
1. `date_tested` - this is not used by newly added test orders 
2. `date_tested_backdate` - this stores the test date from the client

Not sure what the purpose of column 1 is; I don't know how to access the production db and check if all the existing records use `date_tested` to store the wrong times. If they do, this query should be updated to reference that column instead